app/models/volunteer.rb

```diff
@@ -0,0 +1,14 @@
+# CaseContact Model
```

Comment:
This comment is not accurate, this is the Volunteer model. Not a big deal :)

Reply:
Oh oops, too much copy and paste, thanks for catching!